lib/rails/diff/cli.rb

```diff
@@ -27,6 +27,15 @@ def file(*files)
         options[:fail] ? abort(diff) : puts(diff)
       end
 
+      desc "dotfiles", "Compare dotfiles in your repository with Rails' generated versions"
+      def dotfiles
+        dotfiles = Dir.glob("**/.*").reject do |path|
```

Comment:
This isn't exactly what I had in mind. It should be all dotfiles **in the toplevel** dir. It should also include files inside folders like `.github` and similar. 

I'd like to reject files in .gitignore too.

Maybe we should delegate this to Git?

`git ls-files --cached --others --exclude-standard -- '.*'`